client/src/component/ServiceCard/index.js

```diff
@@ -6,36 +6,42 @@ import CardActionArea from '@material-ui/core/CardActionArea';
 import CardContent from '@material-ui/core/CardContent';
 import CardMedia from '@material-ui/core/CardMedia';
 import Typography from '@material-ui/core/Typography';
-import { CardActions } from '@material-ui/core';
+import { CardActions, Container } from '@material-ui/core';
 import useStyles from './style';
 import { BOOK } from '../../utils/router.constant';
 
 const ServiceCard = ({ service }) => {
-  const { img, name, description } = service;
+  const { image, name, description } = service;
   const classes = useStyles();
   const fakeImage =
     'https://www.matrix.com/~/media/images/hair-color-gallery/hair-color-looks/highlights/brown-highlights/brown-highlights-1.jpg';
   return (
-    <Card className={classes.card}>
-      <CardActionArea className={classes.root}>
-        <CardMedia>
-          <img className={classes.image} src={img || fakeImage} alt="service" />
-        </CardMedia>
-        <CardContent>
-          <Typography gutterBottom variant="h5" component="h2">
-            {name}
-          </Typography>
-          <Typography variant="body2" color="textSecondary" component="p">
-            {description.split(' ').splice(0, 11).join(' ')}
-          </Typography>
-        </CardContent>
-        <CardActions>
-          <Link to={`${BOOK}`} className={classes.link}>
-            Book Now !
-          </Link>
-        </CardActions>
-      </CardActionArea>
-    </Card>
+    <Container>
+      <Card className={classes.card}>
+        <CardActionArea className={classes.root}>
+          <CardMedia>
+            <img
+              className={classes.image}
+              src={image || fakeImage}
+              alt="service"
+            />
+          </CardMedia>
+          <CardContent>
+            <Typography gutterBottom variant="h6" component="h4">
+              {name}
+            </Typography>
+            <Typography variant="body2" color="textSecondary" component="p">
+              {description.split(' ').splice(0, 10).join(' ')}
+            </Typography>
+          </CardContent>
+          <CardActions>
+            <Link to={`${BOOK}`} className={classes.link}>
+              Book Now !
+            </Link>
```

Comment:
please make  this link dynamic,
if the user is logged redirect him to the booking page
if the user is not logged redirect him to the signup page